core/commons.yaml

```diff
@@ -246,6 +246,12 @@ components:
         The extent of the features in the collection. In the Core only spatial and temporal
         extents are specified. Extensions may add additional members to represent other
         extents, for example, thermal or pressure ranges.
+
+        The first items in the arrays always describe the overall extent of
```

Comment:
```suggestion
        The first item in the array describes the overall extent of
```

Reply:
This is copied from OGC API. I'd prefer to keep the specs in sync as much as possible for easier alignment in the future. Or you could propose it to them.

Reply:
sounds good. I will make that suggestion to them.

Reply:
I will make the change in the OGC API Features pull request, too.

Reply:
Updated.